### PR TITLE
221 add count token housing security

### PR DESCRIPTION
**Pull request by td928**
WIP branch and pull request for add `_count` to housing security and quality. Sharing the progress on this because we are near deadline and someone might need to pick up this work.

### CODE REVIEW

File: tests/general_indicator_tests/general_indicator_test_helpers.py [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+from aggregate.all_accessors import Accessors
 
+accessors = Accessors()
 
-def get_by_geo():
+
+def get_by_geo(housing_security=False):
+    """housing security parameter is temporary"""
+    if housing_security:
+        accessors_list = accessors.housing_security
```
Comment (td928) [inline tests/general_indicator_tests/general_indicator_test_helpers.py:9]: This is outside the scope of this pull request but when I peek inside the Accessor class implementation. I found the distinction between the two properties a little difficult to understand. Please let me know when there is a moment to chat about it. Thanks!

Comment (td928) [reply tests/general_indicator_tests/general_indicator_test_helpers.py:9]: I think reviewing your opening comments explained it. So please ignore my questions for now!

**Comment by SashaWeinstein** [comment]
In these last commits I add in the new indicators and clean up some unused imports/print statements. I also comment out a block of unused code in `load_aggregated.py`. Figured this was ok because it needs a refactor anyways

**Comment by SashaWeinstein** [comment]
@td928 I can't assign you as reviewer because you opened this PR, but you should be reviewer

**Comment by td928** [comment]
> @td928 I can't assign you as reviewer because you opened this PR, but you should be reviewer

lol about this. I also could not approve because I opened it.

**Comment by SashaWeinstein** [comment]
> > @td928 I can't assign you as reviewer because you opened this PR, but you should be reviewer
> 
> lol about this. I also could not approve because I opened it.

lol shoot. can you review? or do you want to just leave comments in conversation?

File: aggregate/housing_security/income_restricted_units.py [hunk 2]
```diff
@@ -31,7 +33,10 @@ def load_clean_income_restricted():
     source_data["citywide"] = "citywide"
 
     source_data.rename(
-        columns={"Public Housing Units": "nycha_units_count", "RAD Units": "rad_units_count"},
+        columns={
+            "Public Housing Units": "units_nycha_count",
```
Comment (td928) [inline aggregate/housing_security/income_restricted_units.py:37]: the final output looks good!

**Comment by SashaWeinstein** [comment]
Hey @AmandaDoyle, @mbh329 I think this is big enough to need two reviewers? Once it's approved by one of you I can run the housing security export from dev

File: internal_review/housing_security/citywide/income_restricted_units.csv [hunk 3]
```diff
@@ -1,2 +1,2 @@
-citywide,NYCHA_units,RAD_units
+citywide,nycha_units_count,rad_units_count
```
Comment (td928) [inline internal_review/housing_security/citywide/income_restricted_units.csv:1]: just flag this output is outdated but branch produce right results when run

File: aggregate/all_accessors.py [hunk 4]
```diff
@@ -8,23 +10,32 @@
...
+# Three or more maintenance deficiences also left out as I'm not sure where we are getting data moving forward
 # from aggregate.housing_security.three_or_more_maintenance_deficiencies import (
 #     count_units_three_or_more_deficiencies,
 # )
 
-# Three or more maintenance deficiences also left out as I'm not sure where we are getting data moving forward
-# from aggregate.housing_security.evictions_by_city_marshals import (
-#     count_residential_evictions,
-# )
+# Currently count residential evictions is left out as it requires geocoding, can address this later if need be.
```
Comment (AmandaDoyle) [inline aggregate/all_accessors.py:18]: I think this is outdated.  This is the only thing I have feedback on.  Everything else looks fine to me.

**Review by AmandaDoyle — APPROVED** [review]

File: aggregate/all_accessors.py (hunk 4 above)
Comment (SashaWeinstein) [reply aggregate/all_accessors.py:18]: oh great fantastic catch, will update those comments now

**Pull request merged**